Dockerfile

```diff
@@ -67,15 +67,15 @@ LABEL org.opencontainers.image.source = "https://github.com/valhalla/valhalla"
 
 # grab the builder stages artifacts
 COPY --from=builder /usr/local /usr/local
-COPY --from=builder /usr/lib/python3/dist-packages/valhalla/* /usr/lib/python3/dist-packages/valhalla/
+COPY --from=builder /usr/local/lib/python3.12/dist-packages/valhalla/* /usr/local/lib/python3.12/dist-packages/valhalla/
 
 # we need to add back some runtime dependencies for binaries and scripts
 # install all the posix locales that we support
 RUN export DEBIAN_FRONTEND=noninteractive && apt update && \
     apt install -y \
-      libcurl4 libczmq4 libluajit-5.1-2 libgdal32 \
+      libcurl4 libczmq4 libluajit-5.1-2 libgdal34 \
       libprotobuf-lite32 libsqlite3-0 libsqlite3-mod-spatialite libzmq5 zlib1g \
-      curl gdb locales parallel python3-minimal python3-distutils python-is-python3 \
+      curl gdb locales parallel python3-minimal python3-setuptools python-is-python3 \
```

Comment:
Do we need `setuptools`? I know `distutils` was recently deprecated and some of that functionality moved to setuptools, but I think it was only used for that `strtobool` utility function used in `valhalla_build_config`. But that got replaced in #4017

Reply:
probably not then! that was why I was suggesting we test the Python bits but now that you mention it I think thats the only thing I was wondering about